core/src/main/scala/kafka/admin/Throttle.scala

```diff
@@ -0,0 +1,28 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements.  See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License.  You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package kafka.admin
+
+case class Throttle (value: Long, action: PostUpdateAction = NoPostUpdateAction)
```

Comment:
Is there a reason why you used a `PostUpdateAction` class instead of just a function? e.g.:

```scala
case class Throttle(value: Long, postUpdateAction: () => Unit)
```